server/src/main/java/io/druid/server/coordinator/helper/DruidCoordinatorLogger.java

```diff
@@ -259,7 +259,7 @@ public DruidCoordinatorRuntimeParams run(DruidCoordinatorRuntimeParams params)
     final Stream<DataSegment> allSegments = params
         .getDataSources()
         .stream()
-        .flatMap((final DruidDataSource dataSource) -> dataSource.getSegments().stream());
+        .flatMap((ImmutableDruidDataSource dataSource) -> dataSource.getSegments().stream());
```

Comment:
is the type even needed here?

Reply:
IMHO this can be a bit cleaner with something like 

```
params
  .getDataSources()
  .stream()
  .map(DruidDataSource::getSegments)
  .flatMap(Set::stream);
```

Reply:
Removed.